schema/bom-1.6.schema.json

```diff
@@ -678,8 +678,8 @@
           "$ref": "#/definitions/component"
         },
         "manufacture": {
-          "title": "Manufacture",
-          "description": "The organization that manufactured the component that the BOM describes.",
+          "title": "Manufacturer",
+          "description": "The organization that manufactured the CycloneDX document (the \"manufacturer\", although the property is misspelled).\nThis may be different from the manufacturer of the component that the CycloneDX document describes.",
```

Comment:
:exclamation: this would be a semantic change. technically this is a breaking change.